pkg/services/ngalert/schedule/schedule.go

```diff
@@ -3,26 +3,31 @@ package schedule
 import (
 	"context"
 	"fmt"
+	"net/url"
 	"sync"
 	"time"
 
-	"github.com/benbjohnson/clock"
-	apimodels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
-	"github.com/grafana/grafana/pkg/services/ngalert/metrics"
-	"golang.org/x/sync/errgroup"
-
 	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/services/alerting"
+	apimodels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
 	"github.com/grafana/grafana/pkg/services/ngalert/eval"
+	"github.com/grafana/grafana/pkg/services/ngalert/metrics"
 	"github.com/grafana/grafana/pkg/services/ngalert/models"
+	"github.com/grafana/grafana/pkg/services/ngalert/sender"
 	"github.com/grafana/grafana/pkg/services/ngalert/state"
 	"github.com/grafana/grafana/pkg/services/ngalert/store"
 	"github.com/grafana/grafana/pkg/tsdb"
+
+	"github.com/benbjohnson/clock"
+	"golang.org/x/sync/errgroup"
 )
 
 // timeNow makes it possible to test usage of time
 var timeNow = time.Now
 
+// PollingInterval of how often we sync admin configuration.
+var PollingInterval = 1 * time.Minute
```

Comment:
How about more descriptive names?
```suggestion
var AdminConfigPollingInterval = 1 * time.Minute
```